### DISCUSSION TITLE
How to access belongsTo data in ImageColumn?

### DISCUSSION THREAD

**Discussion by jadersbr** [rappasoft/laravel-livewire-tables#913] [Q&A]
Hi, I'm trying to put an image in a column, but, without success.

I can access image data from `Column::make("Cover", "book.cover")`, where book is a belongsTo method of my model, but I can't access image data from  `ImageColumn::make('Cover')->location(fn($row) => $row->book->cover)`, I receive error message **Attempt to read property "cover" on null**. I also tried `ImageColumn::make('Cover')->location(fn($row) => $row->book()->cover)`. What am I doing wrong?

**Comment by tatuananhnb2608** [comment]
did you fix this error?

**Reply by lrljoe** [reply]
Are you experiencing the same issue?  If not, I'll mark this as closed.
